packages/admin/admin/src/appHeader/dropdown/AppHeaderDropdown.tsx

```diff
@@ -16,17 +22,25 @@ export interface AppHeaderDropdownProps extends Omit<AppHeaderButtonProps, "chil
     onOpenChange?: (open: boolean) => void;
 }
 
-const styles = () => {
-    return createStyles<AppHeaderDropdownClassKey, AppHeaderDropdownProps>({
-        root: {
-            height: "100%",
-        },
-        popoverRoot: {},
-        popoverPaper: {
-            minWidth: "var(--comet-admin-app-header-dropdown-item-width)",
-        },
-    });
-};
+const Root = styled("div", {
+    name: "CometAdminAppHeaderDropdown",
+    slot: "root",
+    overridesResolver(_, styles) {
+        return [styles.root];
+    },
+})(
+    css`
+        height: 100%;
+    `,
+);
+
+const PopoverRoot = styled(Popover, {
+    name: "CometAdminAppHeaderDropdown",
+    slot: "popoverRoot",
```

Comment:
I'd call this just `popover`. @jamesricky what do you think? IMO is a breaking change preferable to an inferior API. Also, we're already removing another element (paper)

Reply:
Yes probably makes sense.